strapi/strapi
# feat: add language selector to the blocks code editor

### What does it do?

- Adds a floating select to specify the language of all code blocks in Blocks.
- Sets up the first end to end test for Blocks (because it's impossible to test in unit tests, since JSDom doesn't support contenteditable)
### Why is it needed?

- Users need to know which language a code block uses, so they can use tools like Prism to do syntax highlighting in their frontend
- We'll be able to do syntax highlighting in blocks pretty soon

### How to test

Create a code block in a Blocks editor. The select should only appear when your code block is focused. The default value should be plain text. You should be able to change it. When you do, you should be able to save the change
